### ISSUE TITLE
Extremely outdated install instructions.

### ISSUE THREAD

**Issue by navid-zamani** [gentoo-haskell/hackport#138]
Overlays haven’t been a thing in Gentoo for many years now.  
It’s repos (repositories) nowadays.

Shouldn’t simply enabling the repo via `eselect repository enable haskell` and then emerging hackport be enough to get everything working? (The only question left would then be where the ebuilds go. Aka selecting one of your local repos. … The current instruction of putting `Just (LocalInfo {distfiles_dir = "/var/cache/distfiles", overlay_list = ["/var/db/repos/haskell"], portage_dir = "/var/db/repos/gentoo"})` in ` ~/.hackport/repositories`  seems to not be suited to work with repos, given that it still says `overlay_list` and `portage_dir`. Hackport should be able to read that from `/etc/portage/repos.conf` and /etc/portage/make.conf`. )

I`m unsure if hackport is deprecated or abandoned nowadays and I haven’t heard about it due to having been away from Gentoo since 2017. (But even back then, overlays were already replaced by repos.)

**Comment by hololeap** [comment]
Yes, the line in the readme is quite old (from [13 years ago](https://github.com/gentoo-haskell/hackport/commit/ab0cf4d8b7d85b1db66ec8d13d94c3ea80cda14e)), but it should be understandable even if it's using an old term. Hackport needs to know where to place the ebuilds it generates, which is usually the repo named `haskell`, but possibly not. I'm not sure how it would get this information from `repos.conf` except by guessing. The method of selecting a repo location could probably be improved.

> I`m unsure if hackport is deprecated or abandoned nowadays

I'm not sure where you're getting this idea from. There isn't much activity, but the last commit was only three weeks ago.

**Comment by hololeap** [comment]
Closing due to inactivity.
